controllers/serverTableController.js

```diff
@@ -1,27 +1,65 @@
-var app = angular.module('pokemonDraft', []);
-app.controller('serverTableController', function($scope, $http) {
+var app = angular.module('pokemonDraft', ['ui.bootstrap.modal', 'ui.bootstrap.tpls']);
+app.controller('serverTableController', function($scope, $http, $modal) {
+	// server list
+	$scope.hostedGames = [];
+	$scope.usersID;
```

Comment:
What's your preference on indentation? This is a single tab of 4 spaces in length, personally i prefer spaces to tabs, and only use 2 spaces for an indent. This is heavily personal opinion, try converting one file to 2 space indentation on sublime and see which u prefer, aslong as it's consistant it doesn't matter